test/smoke/testApps/SimpleTelemetryData/src/main/java/com/microsoft/ajl/simplecalc/SimpleCalculatorServlet.java

```diff
@@ -0,0 +1,127 @@
+package com.microsoft.ajl.simplecalc;
+
+import com.microsoft.ajl.simplecalc.model.BinaryCalculation;
+import com.microsoft.ajl.simplecalc.model.BinaryOperator;
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.applicationinsights.telemetry.Duration;
+import com.microsoft.applicationinsights.telemetry.ExceptionTelemetry;
+import com.microsoft.applicationinsights.telemetry.RequestTelemetry;
+import com.microsoft.applicationinsights.telemetry.SeverityLevel;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static com.microsoft.ajl.simplecalc.ParameterConstants.*;
+
+/**
+ * Servlet implementation class SimpleCalculatorServlet
+ */
+@WebServlet(description = "Performs given calculation", urlPatterns = { "/doCalc" })
```

Comment:
Instead of duplicating these each time, you can add another Servlet class to the same test. You shouldn't need to create a new testapp unless you're changing how the app works, e.g. servlet with core only, servlet with a filter, spring boot with filter, jax-rs with agent, etc.

Reply:
@littleaj Thanks for your comments. I will change my code. And I have a question. Do you mean to write more servlets send different track data?

Reply:
Yes, a test app should have multiple endpoints/servlets which correspond to different track data calls.